docs/rules/README.md

```diff
@@ -8,266 +8,265 @@ Some rules are fixable using the `--fix` command line flag. Those rules are mark
+* [no-octal](no-octal.md): disallow octal literals (recommended)
+* [no-octal-escape](no-octal-escape.md): disallow octal escape sequences in string literals
+* [no-param-reassign](no-param-reassign.md): disallow reassigning `function` parameters
+* [no-proto](no-proto.md): disallow the use of the `__proto__` property
+* [no-redeclare](no-redeclare.md): disallow `var` redeclaration (recommended)
+* [no-return-assign](no-return-assign.md): disallow assignment operators in `return` statements
+* [no-script-url](no-script-url.md): disallow `javascript:` urls.
 * [no-self-assign](no-self-assign.md): disallow assignments where both sides are exactly the same (recommended)
 * [no-self-compare](no-self-compare.md): disallow comparisons where both sides are exactly the same
-* [no-sequences](no-sequences.md): disallow use of the comma operator
-* [no-throw-literal](no-throw-literal.md): restrict what can be thrown as an exception
-* [no-unmodified-loop-condition](no-unmodified-loop-condition.md): disallow unmodified conditions of loops
-* [no-unused-expressions](no-unused-expressions.md): disallow usage of expressions in statement position
+* [no-sequences](no-sequences.md): disallow comma operators
+* [no-throw-literal](no-throw-literal.md): disallow throwing literals as exceptions
+* [no-unmodified-loop-condition](no-unmodified-loop-condition.md): disallow unmodified loop conditions
+* [no-unused-expressions](no-unused-expressions.md): disallow unused expressions
 * [no-unused-labels](no-unused-labels.md): disallow unused labels (recommended)
-* [no-useless-call](no-useless-call.md): disallow unnecessary `.call()` and `.apply()`
+* [no-useless-call](no-useless-call.md): disallow unnecessary calls to `.call()` and `.apply()`
 * [no-useless-concat](no-useless-concat.md): disallow unnecessary concatenation of literals or template literals
-* [no-useless-escape](no-useless-escape.md): disallow unnecessary usage of escape character
-* [no-void](no-void.md): disallow use of the `void` operator
-* [no-warning-comments](no-warning-comments.md): disallow usage of configurable warning terms in comments: e.g. `TODO` or `FIXME`
-* [no-with](no-with.md): disallow use of the `with` statement
-* [radix](radix.md): require use of the second argument for `parseInt()`
-* [vars-on-top](vars-on-top.md): require declaration of all vars at the top of their containing scope
-* [wrap-iife](wrap-iife.md): require immediate function invocation to be wrapped in parentheses
-* [yoda](yoda.md): require or disallow Yoda conditions
+* [no-useless-escape](no-useless-escape.md): disallow unnecessary escape characters
+* [no-void](no-void.md): disallow `void` operators
+* [no-warning-comments](no-warning-comments.md): disallow specified warning terms in comments
+* [no-with](no-with.md): disallow `with` statements
+* [radix](radix.md): require the second argument when using `parseInt()`
```

Comment:
It looks the radix rule now has options, so its description needs rewrite for changed meaning:
* probably with “enforce consistent” pattern
* suggest “radix argument” instead of “second argument”